posthog/warehouse/api/external_data_schema.py

```diff
@@ -74,6 +75,12 @@ class Meta:
             "status",
         ]
 
+    def get_status(self, schema: ExternalDataSchema) -> str | None:
+        if schema.status == ExternalDataSchema.Status.CANCELLED:
+            return "Billing limits"
```

Comment:
Is this always the case? What if a user realizes they made a mistake and want to cancel the job? Maybe we should have a separate `CANCELLED_BILLING_LIMIT` status?

Reply:
There's no way to cancel a job right now